ckanext/validation/plugin.py

```diff
@@ -65,236 +38,111 @@ class ValidationPlugin(MixinPlugin, p.SingletonPlugin, DefaultTranslation):
     # ITranslation
     def i18n_directory(self):
         u'''Change the directory of the .mo translation files'''
-        return os.path.join(
-            os.path.dirname(__file__),
-            'i18n'
-        )
+        return os.path.join(os.path.dirname(__file__), 'i18n')
 
     # IConfigurer
 
     def update_config(self, config_):
         if not tables_exist():
-            if is_ckan_29():
```

Comment:
Why is the CKAN 2.8 logic being discarded?

Reply:
I was developing for ckan-2.9.5+, since the 2.8 support wasn't in the requirements. I'm going to restore the code related to 2.8 support.

Reply:
We just moved to 2.9.x  on www.data.qld.gov.au

ckanext-validation needs to be able to be re-synced back to upstream repo. So please keep 2.8 (or even 2.7) compatibility.

Reply:
This will be fixed and pushed by Alex shortly.